imap_processing/codice/codice_l1a.py

```diff
@@ -37,16 +317,70 @@ def codice_l1a(packets: list[space_packet_parser.parser.Packet]) -> str:
         The path to the CDF file that was created
     """
     # Group data by APID and sort by time
-    grouped_data = group_by_apid(packets)
+    print("Grouping the data by APID")
+    # TODO: Turn this on once test data is acquired
+    # grouped_data = group_by_apid(packets)
+    grouped_data = {
+        CODICEAPID.COD_NHK: [],
+        CODICEAPID.COD_LO_SW_SPECIES_COUNTS: [],
+    }  # Temporary during development
 
     for apid in grouped_data.keys():
+        print(f"processing {apid} packet")
         if apid == CODICEAPID.COD_NHK:
             sorted_packets = sort_by_time(grouped_data[apid], "SHCOARSE")
             data = create_dataset(packets=sorted_packets)
-        else:
+
+        elif apid == CODICEAPID.COD_LO_SW_SPECIES_COUNTS:
+            packets = grouped_data[apid]
+            table_id, plan_id, plan_step, view_id = get_params(packets)
+
+            pipeline = CoDICEL1aPipeline(table_id, plan_id, plan_step, view_id)
+            pipeline.get_esa_sweep_values()
+            pipeline.get_acquisition_times()
+            pipeline.get_lo_data_products()
+            pipeline.unpack_science_data()
+
+            data = pipeline.make_cdf_data()
+
+        elif apid == CODICEAPID.COD_LO_PHA:
+            logger.debug(f"{apid} is currently not supported")
+            continue
+
+        elif apid == CODICEAPID.COD_LO_PRIORITY_COUNTS:
+            logger.debug(f"{apid} is currently not supported")
+            continue
+
+        elif apid == CODICEAPID.COD_LO_NSW_SPECIES_COUNTS:
+            logger.debug(f"{apid} is currently not supported")
+            continue
+
+        elif apid == CODICEAPID.COD_LO_SW_ANGULAR_COUNTS:
             logger.debug(f"{apid} is currently not supported")
+            continue
+
+        elif apid == CODICEAPID.COD_LO_NSW_ANGULAR_COUNTS:
+            logger.debug(f"{apid} is currently not supported")
+            continue
+
+        elif apid == CODICEAPID.COD_HI_PHA:
+            logger.debug(f"{apid} is currently not supported")
+            continue
+
+        elif apid == CODICEAPID.COD_HI_OMNI_SPECIES_COUNTS:
+            logger.debug(f"{apid} is currently not supported")
+            continue
+
+        elif apid == CODICEAPID.COD_HI_SECT_SPECIES_COUNTS:
+            logger.debug(f"{apid} is currently not supported")
+            continue
 
     # Write data to CDF
-    cdf_filename = write_cdf(data)
+    # cdf_filename = write_cdf(data)
```

Comment:
If you are going to write cdf file here in the future, don't forget to keep track of files paths as you process multiple apid(s). Because you will need that to upload data from cli.py.